### PR TITLE
Helvetica and spacing

### PR DESCRIPTION
**Pull request by allancorbett**
**Description**
Implements Helvetica and adjusts spacing between objects.

### CODE REVIEW

File: packages/gravity-ui-web/src/sass/05-components/03-organisms/_hero.scss [hunk 1]
```diff
@@ -36,6 +36,7 @@
     }
 
     h1 {
+      max-width: $grav-page-content-max-width - ($container-horizontal-max-margin * 2);
```
Comment (james-nash) [inline packages/gravity-ui-web/src/sass/05-components/03-organisms/_hero.scss:39]: Why is this needed?

Shouldn't headings just have their default `display: block;` behaviour and fill out whatever horizontal space they're given by their parent?

If a certain consumer of Gravity, like the website, need to tweak things they should do so in their code. Otherwise, we might risk breaking things in other consumers - e.g. Cycle 7 website.

File: packages/gravity-ui-web/src/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss [hunk 2]
```diff
@@ -7,7 +7,10 @@ $grav-page-heading-trigger-breakpoint: grav-breakpoint(medium);
 // width ($grav-page-content-max-width) themselves, where necessary.
 body > header,
 .grav-c-page-header {
+  position: sticky;
```
Comment (james-nash) [inline packages/gravity-ui-web/src/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss:10]: Hmm... I'm not sure about doing this here. That would mean any project using Gravity will automatically get a sticky header (and it will be sticky across all screensizes too). Is that really what we want?

If this is intended for the Buildit website, then I'd suggest putting those rules in the website's own CSS. It's fine for downstream projects like the website to tweaks some of Gravity's defaults to suit their specific needs, but I'd be weary of doing something like this in Gravity itself.

That being said, this is not a hard "no". If you feel strongly this is how all Gravity page header should be, then this is the right place to do it.

Comment (james-nash) [reply packages/gravity-ui-web/src/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss:10]: Also, whether its here or in the website's code, don't forget to give the header a solid background colour. Right now, the page content shines through and it becomes unreadable...
image

File: README.md [hunk 3]
```diff
@@ -1,45 +1,53 @@
...
 ## Development
 
 ### Initial setup
+
 After an initial clone, or if any of the packages' dependencies have changed you should run the following **in the repo's root directory**:
 
 ```
 npm install
-npm lerna:bootstrap
+lerna bootstrap
```
Comment (james-nash) [inline README.md:25]: Actually, the instructions were correct as they were.

`lerna bootstrap` would only work if you happen to install Lerna globally, which I'd be reluctant to force people to do (I find global NPM things just clutter up your computer and sometimes lead to weird version conflicts when they differ to a locally installed copy in a project)

File: README.md [hunk 4]
```diff
@@ -1,45 +1,53 @@
...
 ```
 
 This will install dependencies for all packages in the repo and configure any local cross-references between them.
 
+You may need to install [Lerna](https://lerna.js.org/) globally using:
+
+```
+npm lerna:bootstrap
```
Comment (james-nash) [inline README.md:33]: If you revert the bit above, you can get rid of this too.

FWIW, this would just run `lerna bootstrap` - but using the copy of lerna installed into Gravity's `node_modules`. To install lerna globally, you'd need to do `npm install -g lerna`.

File: packages/gravity-ui-web/src/sass/05-components/03-organisms/_hero.scss (hunk 1 above)
Comment (allancorbett) [reply packages/gravity-ui-web/src/sass/05-components/03-organisms/_hero.scss:39]: Correct, removed.

File: packages/gravity-ui-web/src/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss (hunk 2 above)
Comment (allancorbett) [reply packages/gravity-ui-web/src/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss:10]: Correct!

File: README.md (hunk 4 above)
Comment (allancorbett) [reply README.md:33]: 👍
